unstructured/partition/pdf_image/pdf_image_utils.py

```diff
@@ -75,6 +78,38 @@ def convert_pdf_to_image(
     return images
 
 
+def pad_element_bboxes(
+    element: "LayoutElement",
```

Comment:
Not sure this needs to be changed since I'm not seeing an error from it, but why does the type hint have parentheses around it?

Reply:
No error is raised from it. I changed it as a refactoring. Do you mean double quotes("") by the "parentheses"?

Reply:
Whoops. Yes, I meant quotes not parentheses.
Sounds good. Was mostly curious since I hadn't seen that before.

Reply:
The double quotes are used to reference the LayoutElement type with the `TYPE_CHECKING` constant in a way that is compatible with both runtime execution (where LayoutElement is not actually imported) and static type checking (where LayoutElement is available).